examples/modular-transformers/modeling_dummy.py

```diff
@@ -441,7 +420,7 @@ def forward(
         output_attentions: bool = False,
         use_cache: bool = False,
         cache_position: Optional[torch.LongTensor] = None,
-        position_embeddings: Optional[Tuple[torch.Tensor, torch.Tensor]] = None,  # will become mandatory in v4.46
+        position_embeddings: Optional[Tuple[torch.Tensor, torch.Tensor]] = None,
```

Comment:
should be mandatory! 
```suggestion
         position_embeddings: Tuple[torch.Tensor, torch.Tensor]
```